docs/benchmarks/webrtc.md

```diff
@@ -0,0 +1,32 @@
+# WebRTC
+
+The WebRTC tests conducted for the Jellyfish media server aimed to assess its performance and reliability in handling real-time communication scenarios, replicating real-life online meetings. 
+
+## Setup
+
+The benchmarks use an 8-core Intel Xeon Processor (Skylake, IBRS).
+
+The peers publish audio tracks and simulcast video tracks,
+publishing three video encodings of 150, 500 and 1500 Kbps
+and receiving the highest one.
+
+## Benchmark results
+
+We performed tests with different configuration - multiple rooms of size two, 
+one big room and an intermediate option.
+
+The values for CPU and Network usage are averaged over time, however they
+remained stable for the duration of the benchmark.
+
+| Rooms | Peers Per Room | CPU Usage | Network usage | Network usage |
```

Comment:
Just make a distinction between incoming and outgoint network throughput as we do in hls benchmarks